src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -213,6 +217,30 @@ public static SimpleInterval trimInterval(final Locatable interval,
                 dataLine -> toModeledSegment(dataLine, false));
     }
 
+    /** Read only the sample names from a segment file.
+     *
+     * @param segmentFile Never {@code null}
+     * @return a list of the unique sample names in the seg file.
+     */
+    public static List<String> readSampleNamesFromSegmentFile(final File segmentFile) {
```

Comment:
Ah right, forgot to mention that you have to change the signature of readSegmentFile to just return List<T>.  I think this is OK.